Lib/test/clinic.test.c

```diff
@@ -5006,75 +5006,6 @@ PyDoc_STRVAR(new_dest__doc__,
 /*[clinic end generated code: output=9cac703f51d90e84 input=090db8df4945576d]*/
 
 
-/*[clinic input]
```

Comment:
Why this test has been deleted?

Reply:
I've put it back in locally; I mistook it for a duplicate test.